libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -582,12 +583,31 @@ public synchronized String getEncryptionKeyForPasscode(String actualPass) {
     }
 
     /**
+     * Returns the branded login path.
+     *
+     * @return Branded login path, or empty string if not configured.
+     */
+    public String getBrandedLoginPath() {
```

Comment:
I can rename it to `loginBrand`. I don't want it to be confused with community/org level branding - that's something else we don't really support today.